### DISCUSSION TITLE
[Feature Request] Ship new releases with .deb and .rpm packages

### DISCUSSION THREAD

**Discussion by zkkv** [cabinpkg/cabin#1160] [Ideas]
It would be handy to have DEB and RPM binary packages shipped alongside the source code of poac with each release because building from source is rarely fun for users. This could be done via a set of scripts that create the mentioned packages. The scripts can be kept in the source control.

**Comment by ken-matsui** [comment]
This sounds great, although I don't have experience creating these. Anyone with experience is welcome to take it on.

**Comment by ken-matsui** [comment]
Closed https://github.com/cabinpkg/cabin/issues/1206
